build.sh

```diff
@@ -2,6 +2,20 @@
 
 set -e
 
+if [ -d ".git" ]; then
+
+SHA=`git rev-parse --short --verify HEAD`
+DATE=`git show -s --format="%cd" --date="format:%Y-%m-%d" HEAD`
```

Comment:
This apparently is not supported on every `git` version. Any ideas how to nicely find out the commit date?